examples/onnxruntime/pytorch/text-classification/run_glue.py

```diff
@@ -297,6 +297,9 @@ def main():
     else:
         model_args, data_args, training_args, optim_args = parser.parse_args_into_dataclasses()
 
+    if data_args.ort_train:
```

Comment:
@michaelbenayoun Yeah it is not logical to put `ort_train` under `DataArguments`. 
I am thinking of adding `ort_train` to a new `ORTTrainingArguments` class to trigger ORT for training. And the reason why I kept `do_train` is that I need it to trigger PyTorch for training since ORT training is currently not compatible with ORT inference and we might want that for the quantizer or the optimizer part.